docs/conflicts.md

```
@@ -33,24 +33,46 @@ conflicts with the following logic:
 
-To make a column a conflict-free delta-apply column, ensuring that the value
-replicated is the delta of the committed changes (the old value plus or
-minus any new value) to a given record, you need to apply the following
-settings to the column: `log_old_value=true,
-delta_apply_function=spock.delta_apply`. For example:
+To make a column a conflict-free delta-apply column, ensuring that the
+value replicated is the delta of the committed changes (the old value
+plus or minus any new value) to a given record, attach a SECURITY LABEL
+to the column via the `spock.delta_apply()` helper:
 
 ```sql
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)